src/main/resources/application.properties

```diff
@@ -8,17 +8,18 @@ db.maxPoolSize=18
 db.poolIncrement=3
 
 #Mail Config
-spring.mail.host=smtp.gmail.com
-spring.mail.port=587
-spring.mail.username=ims.work@gmail.com
-spring.mail.password=ims.work
-spring.mail.protocol=smtp
+mail.host=smtp.gmail.com
+mail.port=587
+mail.username=ims.work@gmail.com
+mail.password=ims.work
+mail.protocol=smtp
 mail.smtp.starttls.enable=true
 mail.debug=true
 mail.smtp.auth=true
-mail.main.URL=http://localhost:8080
-mail.activationURL=${mail.main.URL}/users/confirmation?emailUUID=
-mail.resetPasswordURL=${mail.main.URL}/reset-password?emailUUID=
+mail.main.url=http://localhost:8080
+mail.ui.url=http://localhost:4200
```

Comment:
spring.base.url=http://localhost:8080
ui.base.url=http://localhost:8080

Reply:
`ui.url` port must be 4200

Reply:
Yeah, I missed it because of copy paste :)
```properties
spring.base.url=http://localhost:8080
ui.base.url=http://localhost:4200
```

Reply:
Okey)